pages/conduct/enforcement-manual.md

```diff
@@ -0,0 +1,60 @@
+<script>{
+	"title": "jQuery Foundation Code of Conduct - Enforcement Manual",
+	"pageTemplate": "page-conduct.php"
+}</script>
+
+This is the enforcement manual followed by jQuery's Code of Conduct Committee. It's used when we respond to an issue to make sure we're consistent and fair. It should be considered an internal document, but we're publishing it publicly in the interests of transparency.
+
+### The Code of Conduct Committee
+
+All responses to reports of conduct violations will be managed by a [Code of Conduct Committee](https://jquery.org/conduct/comittee/) ("the committee").
+
+The jQuery Foundation's Board of Directors ("the board") will establish this committee, compromised of at least three members. One member will be designated chair of the group and will be responsible for all reports back to the board. The board will review membership on a regular basis.
+
+### How the committee will respond to reports
+
+When a report is sent to the committee they will immediately reply to the report to confirm receipt. This reply must be sent within 24 hours, and the group should strive to respond much quicker than that.
+
+See the [reporting guidelines](https://www.jquery.org/conduct/reporting/) for details of what reports should contain. If a report doesn't contain enough information, the committee will obtain all relevant data before acting. The committee is empowered to act on the jQuery Foundation's behalf in contacting any individuals involved to get a more complete account of events.
+
+The committee will then review the incident and determine, to the best of their ability:
+
+*   what happened
+*   whether this event constitutes a code of conduct violation
+*   who, if anyone, was the bad actor
+*   whether this is an ongoing situation, and there is a threat to anyone's physical safety
+
+This information will be collected in writing, and whenever possible the group's deliberations will be recorded and retained (i.e. IRC transcripts, email discussions, recorded voice conversations, etc).
+
+The working group should aim to have a resolution agreed upon within one week. In the event that a resolution can't be determined in that time, the group will respond to the reporter(s) with an update and projected timeline for resolution.
+
+### Acting Unilaterally
+
+If the act is ongoing (such as someone engaging in harassment in #jquery), or involves a threat to anyone's safety (e.g. threats of violence), any working group member may act immediately (before reaching consensus) to end the situation. In ongoing situations, any member may at their discretion employ any of the tools available to the working group, including bans and blocks.
```

Comment:
This is a fantastic idea.  That removes some of the ambiguity and clarifies the "who can do what" - something that I could see being argued by more persistent trolls.